norona_marynicole/countries.js

```diff
@@ -0,0 +1,95 @@
+  if (!countryName) {
+    document.getElementById("countryDetails").innerHTML =
+      "<p>Please enter a country name.</p>";
+    document.getElementById("sameRegionCountries").innerHTML = "";
+    document.getElementById("newCountryDetails").innerHTML = "";
+    return;
+  }
+
+  fetch("https://restcountries.com/v3.1/name/" + countryName)
+    .then(function (response) {
+      if (!response.ok) {
+        throw new Error("Country not found");
+      }
+      return response.json();
+    })
+    .then(function (countryData) {
+      let country = countryData[0];
+      let details = `
+            <h2>Country Details - ${country.name.common}</h2>
+            <img src="${country.flags.svg}" alt="Flag of ${country.name.common}
+                     " width="100">
+            <p><strong>Area:</strong> ${
+              country.area
+                ? country.area.toLocaleString() + " square kilometers"
+                : "N/A"
+            }</p>
+            <p><strong>Population:</strong> ${
+              country.population ? country.population.toLocaleString() : "N/A"
+            }</p>
+            <p><strong>Region:</strong> ${
+              country.region ? country.region : "N/A"
+            }</p>
+            <p><strong>Capital:</strong> ${
+              country.capital ? country.capital[0] : "N/A"
+            }</p>
+            <p><strong>Currency:</strong> ${
+              country.currencies
+                ? Object.values(country.currencies).join(", ")
+                : "N/A"
+            }</p>
+            <p><strong>Calling Code:</strong> ${
+              country.callingCodes ? country.callingCodes[0] : "N/A"
+            }</p>
+            <p><strong>Timezones:</strong> ${
+              country.timezones ? country.timezones.join(", ") : "N/A"
+            }</p>
+            <p><strong>Native Name:</strong> ${
+              country.name.nativeName
+                ? Object.values(country.name.nativeName).join(", ")
+                : "N/A"
+            }</p>
+            <p><strong>Official Languages:</strong> ${
+              country.languages
+                ? Object.values(country.languages).join(", ")
+                : "N/A"
+            }</p>
+          `;
+      document.getElementById("countryDetails").innerHTML = details;
+
+      return fetch("https://restcountries.com/v3.1/region/" + country.region);
+    })
+    .then(function (response) {
+      if (!response.ok) {
+        throw new Error("Region not found");
+      }
+      return response.json();
+    })
+    .then(function (regionData) {
+      let region = regionData[0].region;
+      let sameRegionCountriesList = regionData
+        .map(function (c) {
+          return `
+              <div class="country-card">
+                <img src="${c.flags.svg}" alt="Flag of ${c.name.common}"
+                 width="50">
+                <p>${c.name.common}</p>
+              </div>
+            `;
```

Comment:
Remove unnecessary breaking of lines. Place line breaks in proper places

Follow the guidelines
https://docs.google.com/presentation/d/1iCoz17mBxZtIQ9PifAW0jjKsuv4xDTlx-TPQxdLySow/edit#slide=id.g29e6ce31160_0_27